docs/spark-connector/spark-authentication-with-gravitino.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Spark authentication with Gravitino server"
+slug: /spark-connector/spark-authentication
+keyword: spark connector authentication oauth2 kerberos
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Spark connector supports `simple` `oauth2` and `kerberos` authentication when accessing Gravitino server.
+
+| Property                     | Type   | Default Value | Description                                                                                                         | Required | Since Version |
+|------------------------------|--------|---------------|---------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| spark.sql.gravitino.authType | string | `simple`      | The authentication mechanisms when communicating with Gravitino server, supports `simple`, `oauth2` and `kerberos`. | No       | 0.7.0         |
+
+## Simple mode
+
+In the simple mode, you could specify the username by configuration.
+
+| Property                            | Type   | Default Value | Description                           | Required | Since Version |
+|-------------------------------------|--------|---------------|---------------------------------------|----------|---------------|
+| spark.sql.gravitino.simple.userName | string | None          | The user name to access the Gravitino | No       | 0.7.0         |
```

Comment:
Can we use `spark.sql.gravitino.auth.simple.userName`?

Reply:
use `spark.sql.gravitino.simpleAuth.userName`, WDYT?

Reply:
@jerqi 
Do you have any opinions on this matter? I think `simpleAuth` is okay but `oauth2` and `kerberos` also need to be changed accordingly.

Reply:
Spark user could specify the username by configuration.

Reply:
You can use `getCurrentUser` instead of configuration. Likely, you can get a use conf Spark origin Kerberos configuration.

Reply:
ok